puppet/zulip/files/postgresql/process_fts_updates

```diff
@@ -43,6 +43,8 @@ sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), "../.
 try:
     os.environ['DJANGO_SETTINGS_MODULE'] = 'zproject.settings'
     from django.conf import settings
+    if settings.USING_PGROONGA:
+        exit()
```

Comment:
Sure.
PGroonga supports read lock free updating. It means that search performance isn't decreased while we add new messages. So we don't need to defer full-text search index update.

(I don't know why Zulip defers full-text search index update. So the answer may not describe what you want to know.)